### DISCUSSION TITLE
How to prevent closure of Tabby windows

### DISCUSSION THREAD

**Discussion by tomitomas** [Eugeny/tabby#9967] [Q&A]
Hi there ! 

I'm still amazed with that wonderful tool ! ❤️ 
(_working on windows, v1.0.215_)

I'd love to let tabby windows always open --> is there a way to do so ? 

like :
- disable the "close button"  (top right red cross) ? 
- get a "confirmation message" before closing ? (that way we can avoid the automatism of clicking on the top rigth button :) )
- an option reduce the windows on the (windows) task bar instead of closing
- ....? 

thanks for your hint !

**Comment by ksprs** [comment]
I also was unable to find how to prevent window from accidental closure. This happens when wrongly press on cross (X) at the top right corner.
Please add an confirmation message.
